src/display/editor/tools.js

```diff
@@ -870,6 +871,7 @@ class AnnotationEditorUIManager {
     };
     this.isShiftKeyDown = false;
     this._editorUndoBar = editorUndoBar || null;
+    this._supportsPinchToZoom = supportsPinchToZoom || false;
```

Comment:
Everywhere through-out this patch the default value for this option should instead be `true` such that it's consistent with https://github.com/mozilla/pdf.js/blob/8fa439826632c5e7e2f3671673998a3a4791bea0/web/app_options.js#L129-L133

*Note:* The only reason for having that configurable at all is because it can be disabled via preferences in Firefox (manually in about:config).